File: Java/ArmstrongNumber.java
import java.util.Scanner;

public class ArmstrongNumber 
{
    public static void main(String[] args) 
    {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number : ");
        int n = scanner.nextInt();

        int rem, cube, result = 0;
        int temp = n;

        while (n != 0) 
        {
            rem = n % 10;
            cube = rem * rem * rem;
            result = result + cube;
            n = n / 10;
        }

        if (temp == result) 
        {
            System.out.println(temp + " is an Armstrong number");
        } 
        else 
        {
            System.out.println(temp + " is not an Armstrong number");
        }
    }
}

File: C#/ArmstrongNumber.cs

//Write a c# program to check armstrong number.

using System;
public class armstrongNumber
{
    static void Main()
    {
        int n;
        Console.WriteLine("Enter the number : ");
        n = int.Parse(Console.ReadLine());

        int rem, cube, result = 0;
        int temp = n;

        while(n!=0)
        {
            rem = n % 10;
            cube = rem * rem * rem;
            result = result + cube;
            n = n / 10;
        }

        n = temp;

        if(n==result)
        {
            Console.WriteLine("{0} is an Armstrong number", n);
        }
        else
        {
            Console.WriteLine("{0} is not an Armstrong number", n);
        }
    }
}